### PR TITLE
Small second pass on logs

### PR DESCRIPTION
**Pull request by ewjoachim**
Cf. #102 

### Successful PR Checklist:
- [ ] Tests
- [ ] Documentation (optionally: [run spell checking](https://github.com/peopledoc/procrastinate/blob/master/CONTRIBUTING.rst#build-the-documentation))
- [x] Had a good time contributing? (if not, feel free to give some feedback)

### CODE REVIEW

**Comment by ewjoachim** [comment]
In wonder if each log should be tested...

**Comment by mgu** [comment]
> In wonder if each log should be tested...

I wonder why no linters have seen the issues

File: procrastinate/worker.py [hunk 1]
```diff
@@ -34,11 +34,17 @@ def __init__(
...
-            f"Log start one queues {self.queues}",
-            extra={"action": "start_log_on_queue", "queues": str(self.queues)},
+        if queues:
+            queue_names = f"queues {', '.join(queues)}"
+        else:
+            queue_names = "all queues"
+
+        self.logger.info(
+            f"Starting worker on queues {queue_names}",
+            extra={"action": "start_worker", "queues": queues},
```
Comment (elemoine) [inline procrastinate/worker.py]: You really want to use `queues` rather than `queue_names` here in the extra's?

Comment (ewjoachim) [reply procrastinate/worker.py]: yes, for me the structured log is supposed to be structured, no ?

Comment (elemoine) [reply procrastinate/worker.py]: Ok I see.

**Pull request merged**